src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Manually install the Android agent (Gradle/Android Studio)
+title: Install the Android agent 
```

Comment:
It seems like it's the "mobile monitoring agent" with an Android plugin. should we be more specific and say "Install the mobile agent for Android"?

Reply:
Note to self: update yml file when I get an answer

Reply:
Let's call it "Android agent" since we only have one, just different confs based on the customer environment